test/raven.client.js

```diff
@@ -382,6 +382,24 @@ describe('raven.Client', function () {
       });
     });
 
+    it('should respect shouldSendCallback', function (done) {
+      client = new raven.Client(dsn, {
+        shouldSendCallback: function (data) {
+          return false;
+        }
+      });
+
+      // neither of these should fire, so report err to done if they do
+      client.on('logged', done);
+      client.on('error', done);
```

Comment:
`done` takes an err argument so if anything is passed to it the test fails; lets you easily say "as long as this callback isn't passed an error, we're good".